lib/everypolitician/popolo.rb

```diff
@@ -30,18 +30,27 @@ def organizations
       end
     end
 
-    class People
+    class Collection
       include Enumerable
 
       attr_reader :documents
 
-      def initialize(documents)
-        @documents = documents.map { |p| Person.new(p) }
-      end
-
       def each(&block)
         documents.each(&block)
       end
+
+      def -(other)
+        ids = documents.map { |d| d.id }
+        other_ids = other.documents.map { |d| d.id }
+        different_ids = ids - other_ids
+        different_ids.map { |id| (documents + other.documents).find { |d| d.id == id } }
```

Comment:
Yep, all good points. Fixed in https://github.com/everypolitician/everypolitician-popolo/commit/393a51e96eb44cc642b3a1933f22c504d4e75540.